packages/core/src/parsers.ts

```diff
@@ -28,11 +20,9 @@ export const parseArray: CCSSParser = (input, prop, options) => {
     }
 }
 
-export const mapValue: CCSSParser = (input, prop, options) => {
-    return options.valueMap?.[prop]?.[input] || input
-}
+export const mapValue: CCSSParser = (input, prop, options) => options.valueMap?.[prop]?.[input] || input
 
-export const parseCCSS: CCSSParser = (input, prop, options) => options.__ccss(input)
+export const parseCCSS: CCSSParser = (input, _, options) => options.__ccss(input)
```

Comment:
?

Reply:
the keyword `prop` was never used. It is a convention to have it as `_` when one argument is not needed. I don't know why but VSCode was complaining about it. 🌵

Reply:
I know you renamed it because of that, but I want to keep consistency in naming and name them the same everywhere.

Reply:
Rolled back.